stampede.go

```diff
@@ -123,11 +123,3 @@ func BytesToHash(b ...[]byte) uint64 {
 	}
 	return d.Sum64()
 }
-
-func StringToHash(s ...string) uint64 {
```

Comment:
Why remove this method? I can’t accept the PR that removes this method. Other part of PR looks good tho, I’ll check it in detail when I’m at a computer

Reply:
Oh, I removed the old default function usage; and removed the function as used only there

Reply:
returned to use